package.json

```diff
@@ -112,5 +112,15 @@
     "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
     "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=text-lcov npm run test-node > coverage.lcov",
     "coverage-report": "nyc report"
-  }
+  },
+  "bundledDependencies": [
```

Comment:
What does this do? In addition, I remember on `jsonld-signatures` we needed a `resolutions` section in the `package.json` to enforce resolutions of sub-dependencies. Do we not need that here?

Reply:
This makes sure that the dependencies get packaged when running `npm pack`. No, we do not need a resolution.